src/main/java/org/broadinstitute/hellbender/tools/CountReads.java

```diff
@@ -34,6 +36,9 @@
 public final class CountReads extends ReadWalker {
 
     private long count = 0;
+
+    @ArgumentCollection
+    final public SimpleOutputCollection out = new SimpleOutputCollection();
     @Override
```

Comment:
Leave a space between the new field and the start of the subsequent method, for readability (also applies to the other classes you've modified below).